netsim/templates/provider/libvirt/Vagrantfile.j2

```diff
@@ -1,6 +1,7 @@
 VAGRANT_COMMAND = ARGV[0]
 
 Vagrant.configure("2") do |config|
+  config.ssh.insert_key = true
```

Comment:
We have 21 Vagrant boxes and most of them are not Linux and thus cannot have their SSH key replaced, and you go ahead and change the system default? Congratulations, great thinking.

FWIW, according to Vagrant documentation, the "true" value is equal to the default behavior, so this was completely unnecessary, but great job anyway.

Reply:
I figured it's the system default, so this only makes explicit what's already happening. However, Vagrant has an elaborate merging process for its settings - see https://developer.hashicorp.com/vagrant/docs/vagrantfile - and so we need to make sure it's set to ```true``` for the private key file to exist in the location we expect it to, in case users may override system defaults. I experimented with setting this to ```false```